ocaml/xapi/cert_distrib.mli

```diff
@@ -17,3 +17,14 @@ val go :
   -> unit
 (** Certificates are collected from [from_hosts] and installed on [to_hosts].
  *  On success, new bundles will have been generated on all [to_hosts] *)
+
+type host_cert
+
+val exchange_certificates_with_pool :
+     rpc:(Rpc.call -> Rpc.response Client.Id.t)
+  -> session_id:API.ref_session
+  -> uuid:string
+  -> certificate:string
+  -> host_cert list
+
+val trust_joined_pool : __context:Context.t -> pool_certs:host_cert list -> unit
```

Comment:
Why can't we merge these two functions? Is it because `exchange_certificates_with_pool` requires an 'unverified' rpc, yet `trust_joined_pool` doesn't?

Would it be possible to handle all of this inside cert_distrib.ml as well? (from a high level i expect one function `val distribute_certificates_between_new_joiner_and_pool : unverified_rpc -> __context -> joiner_uuid -> ... etc ... -> unit`)

Reply:
> Why can't we merge these two functions? 

I just didn't think so far ahead while simplifying the code :)

Reply:
It's looking good now! Thanks